pallets/inactivity-tracking/src/lib.rs

```diff
@@ -331,19 +331,26 @@ pub mod pallet {
                 }
             }
             let active_chains = <ActiveContainerChainsForCurrentSession<T>>::get();
+            // Fetching the parathreads for the current session and remove them
+            // from the active chains array. In this way we handle all parathreads
+            // as inactive chains. This solution would only work if a collator either:
+            // - is assigned to one chain only
+            // - is assigned to multiple chains but all of them are parathreads
+            let parathreads_for_session = T::ParaFilter::get_parathreads_for_session();
+            active_chains
+                .clone()
+                .into_inner()
+                .retain(|x| !parathreads_for_session.contains(x));
```

Comment:
This doesn't do anything because of the clone, I hope tests fail

Reply:
My bad updated in https://github.com/moondance-labs/tanssi/pull/899/commits/c06a56bf979f032c62401621a7641e1360b6c97b

Reply:
Also additional test that will fail if parathread collators are processed as inactive in https://github.com/moondance-labs/tanssi/pull/899/commits/99a6a37b625ba4eb69720cbb0b19eb8e054c66bd